### DISCUSSION TITLE
Override HA Discovery to be Fan

### DISCUSSION THREAD

**Discussion by PopRe** [arendst/Tasmota#13182] [Q&A] [answered]
A PWM fan works exactly like a PWM light, however the discovery for homeassistant announce it as a light.

A fan connected to a switch is OK, but there is an option to also announce it as a light

Suggest an option to override an output to be a fan including relays and PWM

**Comment by barbudor** [comment, marked as answer]
When your use case do not match the standard use case, configure HA manually

**Reply by PopRe** [reply]
I have done that, however, HA is moving away from manual configuration and configuring manually means you don't get the advantages of the device management.  

I believe this could be a pretty common case (people controlling fans with Tasmota) so others could benefit from it.
